### ISSUE TITLE
Kotlin 1.7.20 compatible aarch64 artifacts

### ISSUE THREAD

**Issue by StefanOltmann** [touchlab/SQLiter#86]
I use SQLDelight and tried to update to the recent version of 1.5.4 alongside with updating to Kotlin 1.7.20.

I get this error message:
```
> Task :shared:compileAppleMainKotlinMetadata FAILED
w: Could not find "co.touchlab:sqliter-driver-cinterop-sqlite3" in [/Users/sol/IdeaProjects/myapp, /Users/stefan/.konan/klib, /Users/stefan/.konan/kotlin-native-prebuilt-macos-aarch64-1.7.20/klib/common, /Users/stefan/.konan/kotlin-native-prebuilt-macos-aarch64-1.7.20/klib/platform/ios_arm64]
```

It looks like there is no Kotlin 1.7.20 compatible artifact released yet. Can you please provide one?

Please see https://github.com/cashapp/sqldelight/issues/3592 for more details.

EDIT: Looks like it's a problem with aarch64

**Comment by surajsau** [comment]
+1 facing the same issue 👍

**Comment by samhill303** [comment]
We've recently released 1.8.10 artifacts, please check it out and let us know if you have any compatibility issues
